Test&flake8 failure in template management

```
======================================================================

ERROR: test_get_names_returns_empty_without_interactor (pybleau.app.model.tests.test_plot_template_manager.TestPlotTemplateManager)

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/home/travis/.edm/envs/pybleau-test-3.6-pyqt5/lib/python3.6/site-packages/pybleau-0.6.0.dev0-py3.6.egg/pybleau/app/model/tests/test_plot_template_manager.py", line 92, in test_get_names_returns_empty_without_interactor

    names = self.manager._get_names_from_directory()

AttributeError: 'PlotTemplateManager' object has no attribute '_get_names_from_directory'

----------------------------------------------------------------------
```
Also, 
```
$ python ci flake8
pybleau/app/model/plot_template_manager.py:63:1: W391 blank line at end of file
```

--- a/pybleau/app/model/plot_template_manager.py
+++ b/pybleau/app/model/plot_template_manager.py
@@ -60,4 +60,3 @@ class PlotTemplateManager(HasStrictTraits):
             if filename.endswith(ext):
                 result.append(Path(filename).stem)
         return result
-

--- a/pybleau/app/model/tests/test_plot_template_manager.py
+++ b/pybleau/app/model/tests/test_plot_template_manager.py
@@ -89,5 +89,5 @@ class TestPlotTemplateManager(TestCase, UnittestTools):
         names = ["temp1", "temp2"]
         self.assertCountEqual(names, self.manager.names)
         self.manager.interactor = None
-        names = self.manager._get_names_from_directory()
+        names = self.manager._template_names_from_dir()
         self.assertCountEqual(names, [])

